### ISSUE TITLE
new-branch throws error on pull

### ISSUE THREAD

**Issue by bbergstrom** [sportngin/octopolo#48]
The pull is happening when there is no upstream set.  
```
brianbergstrom@bb-mbp octopolo (master)$ op new-branch dated-branch-delete-flag
git branch --no-track dated-branch-delete-flag origin/master
git checkout dated-branch-delete-flag
git pull
Unable to perform 'git pull': exit_status=1; stderr=There is no tracking information for the current branch.
Please specify which branch you want to merge with.
See git-pull(1) for details

    git pull <remote> <branch>

If you wish to set tracking information for this branch you can do so with:

    git branch --set-upstream-to=origin/<branch> dated-branch-delete-flag

git push --set-upstream origin dated-branch-delete-flag
Branch dated-branch-delete-flag set up to track remote branch dated-branch-delete-flag from origin.
```

**Comment by bbergstrom** [comment]
Fixing in https://github.com/sportngin/octopolo/pull/47

**Comment by anfleene** [comment]
This was resolved by #47
